microceph/api/services.go

```diff
@@ -108,16 +109,28 @@ func cmdRestartServicePost(s *state.State, r *http.Request) response.Response {
 	return response.EmptySyncResponse
 }
 
-func cmdRGWServiceDelete(s *state.State, r *http.Request) response.Response {
-	var req types.RGWService
+// cmdDeleteService handles service deletion.
+func cmdDeleteService(s *state.State, r *http.Request) response.Response {
+	which := path.Base(r.URL.Path)
+	_, ok := ceph.GetConfigTableServiceSet()[which]
+	if !ok {
+		err := fmt.Errorf("%s is not a valid ceph service", which)
```

Comment:
there's quite a bit of this pattern:

```
error = fmt.Errorf(...)
logger.Errorf("%v", error)
```

repeated throughout the code. I think it could be worthwhile factoring out into a utility function.

Reply:
Looked a bit into this, its unfort. not straightforward to make logger support the same formatting directives as Errorf()  specifically the %w error wrapping -- I'll punt on that for now